pages/common/mycli.md

```diff
@@ -13,3 +13,11 @@
 - Connect to a database on the specified host with the specified user:
 
 `mycli -u {{user}} -h {{host}} {{database_name}}`
+
+- Connect to a database with URI:
+
+`mycli mysql://{{user}}@{{host}}`
+
+- Connect to a database with URI more directly:
+
+`mycli mysql://{{user}}:{{password}}@{{host}}:{{port}}/{{database_name}}`
```

Comment:
Same as before. Also, passwords should not be passed in the command line. The command will ask for password and the user should enter. Please see our pages for `psql` and `mysql`. We should try to keep this page consistent with others.

Reply:
As before, If the URI is like saved in project development config file and password is required, Then the password don't have to sit in separate line.

Reply:
Ideally, in settings like that you can pass the password in an environment variable or through something like `.pgpass`. You should almost never set password plainly like this.